packages/core/src/api/formatConversions/simplifyBlocksRehypePlugin.ts

```diff
@@ -22,6 +22,19 @@ export function simplifyBlocks(options: SimplifyBlocksOptions) {
   ]);
 
   const simplifyBlocksHelper = (tree: HASTParent) => {
+    // Checks whether blocks in the tree are wrapped by a parent `blockGroup`
```

Comment:
why is this new / we didn't have this earlier?

Reply:
We were using it a bit differently before. Before, we were calling `unified.process` with this HTML:

```ts
<div data-node-type="blockGroup">
  <div data-node-type="blockContainer">
    ...
  </div>
  <div data-node-type="blockContainer">
    ...
  </div>
  ...
</div>
```

Notice how the `blockContainer` elements aren't wrapped in a `blockGroup` element. But we ideally want to just use `serializeProseMirrorFragment(...)` to get the internal HTML that we pass to `unified.process`, which looks like this: